include/seqan3/io/detail/record.hpp

```diff
@@ -135,8 +136,8 @@ auto const & get_or_ignore(record<field_types, field_ids> const & r)
 }
 
 //!\copydoc seqan3::detail::get_or_ignore
-template <size_t i, typename ...types>
-auto & get_or_ignore(std::tuple<types...> & t)
+template <size_t i, template <tuple_like ...types_> typename tuple_like_t, typename ...types>
```

Comment:
So the this should be a tuple of tulles?

Reply:
No it is not a tuple of tuples. The function was only valid for `std::tuple` as input argument and I want to extend it for seqan3::common_pair. This is needed when we call for instance

```cpp
fout = seqan3::views::zip(ids, seq_quals);
```

Then parameter `t` has type `seqan3::common_pair<string, vector<qualified>>`.

Instead of writing two additional overloads I generalised it to `tuple_like`.